### ISSUE TITLE
[accessibility issue] Panel-dropdown aria labelling could be improved

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#79514]
**Steps to reproduce**:
Go a Grafana Panel on a dashboard and inspect the markup for the panel dropdown options.

The Inspect parent menuitem has a caret icon with a text alternative of "SubMenu icon", however this icon should be marked decorative. The parent menuitem can be indicated as opening a submenu through using aria-haspopup="menu" or "true" and aria-expanded to indicate when the child menu is visible/not visible. Please see 4.121 Name, Role, Value.

**Remediation steps**
Remove aria-label="SubMenu icon" and ensure that ARIA menus follow the [ARIA Authoring Practices Guide design pattern for menus](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/).

![A screenshot of the dashboard panel in a Grafana instance with Chrome devtools open showing the markup with the incorrect aria attributes.](https://github.com/grafana/grafana/assets/6906380/44d4f2b6-e6a7-463e-a30e-82775f2db988)

UHG Audit Id: 1.116

**Comment by tskarhed** [comment]
This has been fixed with a migration from aria-label to test-id. #36523
![image](https://github.com/grafana/grafana/assets/1438972/f42cebdc-f8c6-4471-bb83-3811f3ba84aa)
